website/modules/home-simplified/views/page.html

```
@@ -0,0 +1,49 @@
+
+    <div class="ca-hero">
+      <div class="eyebrow"><span class="pulse"></span>The New Site Is On Its Way</div>
+      <h1>Speed &amp; Function is an engineering and AI consultancy. For over a decade we've partnered with purpose-driven companies to <span class="grad">design, build, and scale software products</span> that carry their mission further.</h1>
+
+      <div class="ca-cue">
+        <span class="cue-txt">Here's what matters most while<br>the new site takes shape.</span>
+      </div>
+
+      <form class="ca-search" id="search-form">
+        <svg class="sicon" width="22" height="22" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2.2" stroke-linecap="round"><circle cx="11" cy="11" r="7"></circle><line x1="21" y1="21" x2="16.2" y2="16.2"></line></svg>
+        <input type="text" placeholder="Find case studies, testimonials, and more">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)